build/ci/github_actions/wait_for_k8s_ready.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -xe
+set +o pipefail
+
+are_pods_ready (){
+  pods=$@
+  for pod in $pods; do
+    running_containers_count=`echo $pod | awk -F / '{print$1}'`
+    total_containers_count=`echo $pod | awk -F / '{print$2}'`
+    if [ $running_containers_count != $total_containers_count ]; then
+      echo true
```

Comment:
why ready is true when not all containers are running?

Reply:
done